react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -29,6 +29,11 @@ type Props = {
      */
     _bottomSheetStyles: StyleType,
 
+    /**
+     * Whether the chat feature has been disabled. The meeting info button will be displayed in its place.
+     */
+    _chatDisabled: boolean,
```

Comment:
Can you please rename it to `_chatEnabled` ? It's a bit confusing going back and forth negating things. I mean here the `!` and `=== false` below on the props mapping.

Reply:
Can do.